Products/CMFCore/FSDTMLMethod.py

```diff
@@ -183,8 +184,9 @@ def setCacheNamespaceKeys(self, keys, REQUEST=None):
     def validate(self, inst, parent, name, value, md=None):
         return getSecurityManager().validate(inst, parent, name, value)
 
-    security.declareProtected(FTPAccess, 'manage_FTPget')
-    manage_FTPget = DTMLMethod.manage_FTPget.im_func
+    if bbb.HAS_ZSERVER:
+        security.declareProtected(FTPAccess, 'manage_FTPget')
+        manage_FTPget = DTMLMethod.manage_FTPget.im_func
 
```

Comment:
We should probably switch to using `__func__`, rather than `im_func`, for forward compatibility with Py3k.